sub-logs/voidedhost.json

```diff
@@ -0,0 +1,15 @@
+{
+    "owner": {
+        "username": "jefrygiglegroder",
+        "email": "mrlagg2much@gmail.com"
+    },
+
+    "target": {
+        "AAAA": {
+            "name": "voidedhost",
+            "value": ["192.168.1.245"]
```

Comment:
That's a local IP address. Not a valid IP.

Reply:
Additionally invalid record type, AAAA is for IPV6.